src/contracts/Repository/Values/Content/URLWildcard/Query/Criterion/Type.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Core\Repository\Values\Content\URLWildcard\Query\Criterion;
+
+/**
+ * Matches URLWildcards based on type.
+ */
+final class Type extends Matcher
+{
+    /**
+     * @var bool
+     */
+    public $isDirect;
+
+    public function __construct(bool $isDirect)
```

Comment:
Hmm, maybe it would be better if this naming was consistent with the existing API (though I like this name)
See
```php
public function create(string $sourceUrl, string $destinationUrl, bool $forward = false): UrlWildcard;
```
Looks like `$forward` is what you want to handle here, right?